src/librustc_target/abi/mod.rs

```diff
@@ -1033,4 +1071,89 @@ impl<'a, Ty> TyLayout<'a, Ty> {
             Abi::Aggregate { sized } => sized && self.size.bytes() == 0
         }
     }
+
+    /// Determines if this type permits "raw" initialization by just transmuting some
+    /// memory into an instance of `T`.
+    /// `zero` indicates if the memory is zero-initialized, or alternatively
```

Comment:
```suggestion
    /// `zero` indicates if the memory can be zero-initialized, or alternatively
```

Reply:
No, I mean "is". Like, you literally have some memory and want to call this before transmuting it, and if you know it to be zero-initialized when set that to `true`.